tools/ci_build/github/windows/setup_env_trt.bat

```diff
@@ -1,2 +1,2 @@
-set PATH=C:\azcopy;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.1\bin;C:\local\cudnn-11.4-windows-x64-v8.2.2.26\cuda\bin;%PATH%
+set PATH=C:\azcopy;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.4\bin;C:\local\cudnn-11.4-windows-x64-v8.2.2.26\cuda\bin;%PATH%
```

Comment:
can you revert this setup_env_trt.bat change for now?
this will allow the unit tests to pass.

Reply:
Ah! I think I got it. This TRT bat is for TRT pipelines. And the windows TRT build machine may not have the latest driver. (I haven't checked). Then it would be ok if you use 11.1 on it. But once you change it to 11.4, it could cause problems.

Reply:
@oliviajain , if you have time, you can make a quick change and set the machine pool to 'Win-GPU-2019', and queue a build. While you are waiting the result, you can follow @jywu-msft 's advice doing the other part.

Reply:
I apologize I didn't make it clear.

Reply:
> Ah! I think I got it. This TRT bat is for TRT pipelines. And the windows TRT build machine may not have the latest driver. (I haven't checked). Then it would be ok if you use 11.1 on it. But once you change it to 11.4, it could cause problems.

FYI. it's not about the driver. there's an issue with using CUDA 11.4 on Windows which causes 2 unit tests to fail. we reported to Nvidia. They reproduced the issue and are looking into it.